.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: guzongmin
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+4. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If applicable, add logs to help explain your problem.
+
+**Environment (please complete the following information):**
+- HW: [e.g. SGX1, SGX2] 
+- OS: [e.g. Ubuntu18.04, CentOS8.1]
+ - Occlum version [e.g. 0.16.0]
+
+**Additional context**
+Add any other context about the problem here.
+
+**Possible Solution/Implementation**
```

Comment:
The titles are not capitalized consistently. Some are capitalized (e.g., `To Reproduce`), while some do not seem to be capitalized (such as `Additional context`). I think it is ok to choose not capitalizing titles in informal writing. But one should not capitalize some tiles, while ignoring the others. Make a choice about the style and stick to it.

If you decide to capitalize the titles, I recommend using [this online tool](https://capitalizemytitle.com/) to automatize the task.